### DISCUSSION TITLE
配置 `adjustEdgeStartAndEnd` 没有生效

### DISCUSSION THREAD

**Discussion by ParadiseWitch** [didi/LogicFlow#1328] [Q&A] [answered]
### 问题
`adjustEdgeStartAndEnd` 配置设置为false时，图中圈出来的位置仍然可以拖动调节（图中指针显示拖动指针）：

![A5ZQHJ GF (U7G`~ TZ`BYN](https://github.com/didi/LogicFlow/assets/37146904/c62cd22a-1ac5-4a06-88c3-12e389e47b8e)

### 代码
<details><summary>Details</summary>
<p>

```js

import LogicFlow from "@logicflow/core";
import "@logicflow/core/dist/style/index.css";

const lf = new LogicFlow({
  container: document.querySelector("#app"),
  grid: true,
  adjustEdgeStartAndEnd: false
});

lf.render({
  nodes: [
    {
      id: "1",
      type: "rect",
      x: 100,
      y: 100,
      text: "矩形1"
    },
    {
      id: "2",
      type: "rect",
      x: 300,
      y: 300,
      text: "矩形2"
    }
  ]
});

```

</p>
</details> 

### 最小复现Demo
https://codesandbox.io/s/logicflow-base2-forked-4sw749?file=/src/index.js:0-429

**Comment by wumail** [comment, marked as answer]
| 属性名                  | 类型    | 默认值 | 说明                                                                   |
| ----------------------- | ------- | ------ | ---------------------------------------------------------------------- |
| adjustEdge              | boolean | true   | 允许调整边                                                           |
| adjustEdgeMiddle        | boolean | false  | 只对折线生效，只允许调整边的中间线段，不允许调整与起点终点相连的线段 |
| adjustEdgeStartAndEnd   | boolean | false  | 允许调整边起点/终点    

当adjustEdgeStartAndEnd为true时，连线的起点和终点分别会有一个可以通过拖拽来调整连线的点
图中的拖拽调整边的是中间线段，如果希望边不能被调整，设置`adjustEdge: false`
